typings/@chainlink/index.d.ts

```diff
@@ -65,7 +65,9 @@ declare module '@chainlink/types' {
     config: Config,
   ) => Promise<AdapterResponse>
 
-  export type ExecuteFactory = (config?: Config) => Execute
+  export interface ExecuteFactory<C extends Config> {
```

Comment:
Any specific reason why you used an fn interface vs an fn type? Interfaces are more powerful, but in the case of functions, fn types read so much better :)

Reply:
Just habit, I see what you mean :smile: